idaes/models_extra/column_models/condenser.py

```diff
@@ -243,9 +243,27 @@ def build(self):
 
         self._make_ports()
 
+        @self.Expression(doc="Split fraction calculation calculations")
+        def reflux_split_fraction(self):
+            return self.reflux_ratio * (1 + self.reflux_ratio) ** -1
+
         if self.config.condenser_type == CondenserType.totalCondenser:
 
-            self._make_splits_total_condenser()
+            make_phase_split(
+                self.control_volume,
+                port=self.reflux,
+                phase=self._liquid_set,
+                side_sf=(self.reflux_split_fraction),
```

Comment:
Minor: why is side_sf value in parenthesis?

Reply:
deleted parenthesis